### ISSUE TITLE
User guide: upgrading tidb-operator itself

### ISSUE THREAD

**Issue by gregwebs** [pingcap/tidb-operator#199]
The user guide explains how to upgrade the tidb cluster. Can we document the procedure for updating tidb-operator? This seems like it should be easy enough if we maintain the right kind of backwards compatibility?

**Comment by tennix** [comment]
Yes, that's right. In previous versions, we changed the labels, this is a break change so upgrading would involve manual intervention. In future versions, we'll keep backward compatibility as much as we can.

I'll update the user guide to add instructions of how to upgrade tidb-operator itself later.
